docs/content/manual/pre-release/upgrade/backing-image-during-upgrade.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Test Backing Image during Longhorn upgrade
+---
+
+1. Set `Concurrent Automatic Engine Upgrade Per Node Limit` to a positive value to enable volume engine auto upgrade.
```

Comment:
Should we add the comment that `Concurrent Automatic Engine Upgrade Per Node Limit` is available from Longhorn v1.1.1 onwards, user upgrading from previous version should set the value while upgrading the Longhorn system?